src/screens/token/components/delegate-token/components/tabs/account-tab/account-tab.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { View } from 'react-native';
+import stylesProvider from './styles';
+import { IThemeProps, withTheme } from '../../../../../../../core/theme/with-theme';
+import { smartConnect } from '../../../../../../../core/utils/smart-connect';
+import { Blockchain, ChainIdType } from '../../../../../../../core/blockchain/types';
+import { AccountAddress } from '../../../../../../../components/account-address/account-address';
+import { IAccountState, ITokenState } from '../../../../../../../redux/wallets/state';
+import { IReduxState } from '../../../../../../../redux/state';
+import { getAccount } from '../../../../../../../redux/wallets/selectors';
+import { connect } from 'react-redux';
+import { StatsComponent } from '../../stats-component/stats-component';
+import { getBlockchain } from '../../../../../../../core/blockchain/blockchain-factory';
+import { getChainId } from '../../../../../../../redux/preferences/selectors';
+import { DelegationCTA } from '../../../../../../../components/delegation-cta/delegation-cta';
+import { Button } from '../../../../../../../library';
+import { translate } from '../../../../../../../core/i18n';
+import { NavigationService } from '../../../../../../../navigation/navigation-service';
+
+export interface IProps {
+    accountIndex: number;
+    blockchain: Blockchain;
+    token: ITokenState;
+}
+
+export interface IReduxProps {
+    account: IAccountState;
+    chainId: ChainIdType;
+}
+
+export const mapStateToProps = (state: IReduxState, ownProps: IProps) => {
+    return {
+        account: getAccount(state, ownProps.accountIndex, ownProps.blockchain),
+        chainId: getChainId(state, ownProps.blockchain)
+    };
+};
+
+export class AccountTabComponent extends React.Component<
+    IProps & IReduxProps & IThemeProps<ReturnType<typeof stylesProvider>>
+> {
+    public render() {
+        const styles = this.props.styles;
+
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+        const stats = blockchainInstance.getStats(this.props.chainId);
```

Comment:
try no to do heavy computation in render functions, render function can be called many times. In this case stats will  have only async methods, so we need to use hooks, or transform this in class and use lifecycle methods.